### PR TITLE
Prevent GoogleTestRunner to execute other TestHosts

### PR DESCRIPTION
**Pull request by noizefloor**
We are currently migrating out Testhosts to Google test. We are still having lots of CPPUnit Testhosts, which are also using "Test" in the executable name. That means these executables are also discovered by GoogleTestRunner as a Google Testhost and are executed in order to discover tests and test suits. To prevent this from happening made some changes so that the discovered executables are now called with parameter "-h". Only if the parameter "--gtest_list_tests" is documented the executable gets treated as a google test host.

### CODE REVIEW

File: GoogleTestRunner/GoogleTestDiscoverer.fs [hunk 1]
```diff
@@ -9,6 +9,7 @@ module Constants =
     [<Literal>]
     let identifierUri = "executor://GoogleTestRunner/v1";
     let gtestListTests = "--gtest_list_tests"
+    let gtestHelp = "-h"
```
Comment (visuve) [inline GoogleTestRunner/GoogleTestDiscoverer.fs:12]: I think `--help` or something else would be better

Comment (noizefloor) [inline GoogleTestRunner/GoogleTestDiscoverer.fs:12]: --help would be fine with me as well.

Comment (visuve) [inline GoogleTestRunner/GoogleTestDiscoverer.fs:12]: Also, what will happen with versions prior to 1.3.0?

https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#running-test-programs-advanced-options

**Comment by noizefloor** [comment]
I just found [GoogleTestAdapter](https://github.com/csoltenborn/GoogleTestAdapter.git). It's pretty much a port of this The Google Test Runner in C#, which is already way better for me. Also it let's you configure the pattern to discover Test binaries. You can do that globally (Tools->Options->Google Test Adapter) or by using a configuration file for every solution. That means you can leave the global setting empty and configure the explicit test host name in the configuration file. Exactly what I was looking for.

**Pull request closed without merging**